gufe/visualization/mapping_visualization.py

```diff
@@ -1,7 +1,9 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/gufe
-from typing import Any, Collection, Optional
+from typing import Any, Collection, Optional, List
 from itertools import chain
+from PIL import Image, ImageOps
```

Comment:
I believe pillow now replaces PIL, which should be used instead.

Additionally - we would need to work out what we want to do with this additional dependency. Probably would need to be an optional.

Reply:
If we use pillow I think rdkit has that as part of its recipe.